templates/create-branches-lagom.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Meta comment: please rename the `templates` folder into `scripts`.

Reply:
Also, if `fork.sh` and `create-branches.sh` are utility files I think we can rename them or move them to an inner folders (e.g. `lib-forking.sh`, `lib-branching.sh` or `libs/...`).

I like the reorganization you've done.